src/IRsend.cpp

```diff
@@ -20,10 +20,27 @@
 // sending IR code on ESP8266
 
 // IRsend ----------------------------------------------------------------------
-
-IRsend::IRsend(uint16_t IRsendPin) {
+// Create an IRsend object.
+//
+// Args:
+//   IRsendPin:  Which GPIO pin to use when sending an IR command.
+//   inverted:   *DANGER* Optional flag to invert the output. (default = false)
+//               e.g. LED is illuminated when GPIO is LOW rather than HIGH.
+//               Setting this to something other than the default could
+//               easily destroy your IR LED if you are overdriving it.
+//               Unless you *REALLY* know what you are doing, don't change this.
+// Returns:
+//   An IRsend object.
+IRsend::IRsend(uint16_t IRsendPin, bool inverted) {
   IRpin = IRsendPin;
   periodOffset = PERIOD_OFFSET;
+  if (inverted) {
+    outputOn = LOW;
```

Comment:
I was going to use ledOn & ledOff, but I thought that would be bad because of the function called ledOff(). :-/